### ISSUE TITLE
Creating the PWM code for the Arduino

### ISSUE THREAD

**Issue by DavidDeu** [linhartr22/count#1]
Hey linhartr22,

thank you very much for this code. How exactly did you used Audacity to convert the sample wave files into PWM? I tried it with my own recorded voice, but unfortunately I failed. So everything else is clear. Just how exactly did you produced these "0x7f,0x7f,0x7f,0x7f,0x7f,..." inputs.

Thank you very much
Best regards
Dave

**Comment by linhartr22** [comment]
The only place I strayed from the article linked in the README was creating the MP3 file in Audacity. I used this Processing application: https://github.com/damellis/EncodeAudio to generate the hex codes. It executes in the Processing IDE that you can download here: https://processing.org/. The Processing IDE will look very similar to the Arduino IDE. I don't remember any challenges getting it working. The EncodeAudio app puts the hex data in the clipboard which you then paste into your .h files. 

This was over six year ago and my memory is a little foggy on the details. Please let me know how it goes or if you have any other questions. I'll do my best to answer them.

**Comment by DavidDeu** [comment]
Hey Linhartr22,

thank you very much for your help. I installed Processing and executed the skript and got this error message:

The method selectInput(String, String) in the type PApplet is not applicable for the arguments (String)

Can you help me how i can select the file i want to convert? I found some Infos in 
https://processing.org/reference/selectInput_.html 
but i couldn't get to the chooser dialog :/
If i add two string inputs in the function selectInput, the error message is:

cannot convert from void to String

I thank you for your help and wish you a nice evening

Best regards
Dave

**Comment by linhartr22** [comment]
I duplicated you error. The first thing I tried was to replace the selectInput with a string constant containing the path to the sample file I just created with Audacity (took me a while to remember how to do that too). Doing so resulting in a new error probably with one of the libraries that aren't support in the current Java version.

I've tried everything I can think of to get the EncodeAudio Processing sketch to work. I tried a couple of version 2 packages I had. Based on the date of then EncodeAudio repo it could have even been designed for version 1 of Processing. That along with potentially requiring an older version of Java, I'm not going to be able to help with this. You might try reaching out to the developer of the EncodeAudio sketch or look for a Processing forum that might be able to help.
